openapi.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.0
+info: 
+  title: To-do API
+  description: To-do app api
+  version: '1.0'
+  
+paths: 
+  /to-dos:
+    get:
+      summary: Return list of to-do
+      responses:
+        '200':
+          description: A Json array of to-dos
+          content: 
+            application/json:
+              schema:
+                type: array
+                items:
+                  content: 
+                  $ref: '#/components/schemas/to-do'
```

Comment:
Burada kebap case kullanmak yerine PascalCase kullanman lazım bu modelleri Go ya generate ederken problem yaşabiliriz.

Reply:
pathlerde de PascalCase e geçirmem gerekli mi /to-dos | /toDos gibi

Reply:
pathlerde olabildigince tek kelime kullanmak lazim aslinda o case muhabbeti ile isimiz olmamali /todos bence, birbirine bagli iki sey varsa user todo lari gibi user/todos gibi mantiksal olarak bolebiliriz  daha genel olan parca solda olur boylece mesela userlarin bloglari olursa
```
user/todos
user/blogs
```
seklinde daha anlamli endpointlerimiz olmus olur